web_widget_timepicker/static/src/js/web_widget_timepicker.js

```diff
@@ -55,35 +55,35 @@ odoo.define('web_widget_timepicker', function (require) {
                 height: height,
                 width: width
             });
-        },                        
+        },
         store_dom_value: function () {
             if (!this.get('effective_readonly')) {
                 this.internal_set_value(
                     this.parse_value(
-                        this.$('input').val(),''));
+                        this.$('input').val(), ''));
             }
         },
-        parse_value: function(val, def) {
-            return formats.parse_value(val, {"widget": this.internal_format}, def);  
+        parse_value: function (val, def) {

```

Comment:
Remove space between function and (val, def) (preserve coding style accross file)